ui/packages/components/src/RunsPage/RunsPage.tsx

```diff
@@ -224,6 +224,19 @@ export function RunsPage({
     [batchUpdate, scrollToTop]
   );
 
+  const onSearchChange = useCallback(
+    (value: string) => {
+      scrollToTop();
+      if (value.length > 0) {
+        const encodedValue = encodeURIComponent(value);
```

Comment:
URI encoding should already happen within `useSearchParam`. Is it not working?

Reply:
I didn't remember their API already included it!